breathecode/marketing/tests/mixins/marketing_test_case.py

```diff
@@ -86,9 +87,20 @@ def check_email_contain_a_correct_token(self, lang, dicts, mock, model):
         self.assertToken(token)
         self.assertTrue(link in html)
 
-    # This function is incompleted
     def check_old_breathecode_calls(self, mock, model):
         self.assertEqual(mock.call_args_list, [
+            call('POST',
+                 'https://old.hardcoded.breathecode.url/admin/api.php',
+                 params=[('api_action', 'contact_sync'),
+                         ('api_key', model['active_campaign_academy'].ac_key),
+                         ('api_output', 'json')],
+                 data={
+                     'email': 'pokemon@potato.io',
```

Comment:
Is prefer split this calls en three functions and make sure every function that use the old breathecode api mock, it is clean in each test in the end of test method